core/domain/draft_upgrade_services_test.py

```diff
@@ -120,6 +120,19 @@ def test_convert_to_latest_schema_version_implemented(self):
             msg='Current schema version is %d but DraftUpgradeUtil.%s is '
             'unimplemented.' % (state_schema_version, conversion_fn_name))
 
+    def test_convert_states_v32_dict_to_v33_dict(self):
```

Comment:
There should be a test for what happens if there are already existing draft changes to the MultipleChoiceInput interaction in the changelist.

Reply:
@seanlip I am not sure what to test or what to change because in ```_convert_states_v32_dict_to_v33_dict()``` I am just returning the value without any change.

Reply:
If a creator has made a draft change that includes the multiple-choice interaction, then their draft changelist would contain that change. It would therefore be in a format that is not compatible with the changes that you are introducing here. Your migration does not appear to be handling this case properly. (See other draft changelist migrations for examples of what I'm talking about.)

Reply:
Done.